docs/architecture/adr-005-enabled-snap-shot-by-default.md

```diff
@@ -0,0 +1,98 @@
+# ADR 005: Enabled snapshot by default
+
+## Changelog
+
+- September 22th, 2021: Initial draft;
+- September 23th, 2021: Moved from DRAFT to PROPOSED
+
+## Status
+
+PROPOSED
+
+## Abstract
+
+Currently, most of the validators or full nodes does not have snapshot enabled because they use the default 
+setting whose snapshot is not enabled. This makes state sync a new node very difficult. 
+In oder to increase the number of snapshots in the network so that making creating a new node easily through 
+`StateSync`, we SHOULD enable snapshot by default.  
+
+## Context
+
+Tendermint provides an alternative mechanism called `StateSync` for bootstrapping a new node without downloading 
+blocks or going through consensus. It fetches a snapshot of the state machine at a given height and restores it.
+`StateSync` will greatly improve the experience of joining a network, reducing the time required to sync a node
+by several orders of magnitude.
+However, state sync a node is almost impossible now since there is too few snapshot in the network now. In order to ensure 
+the number of snapshots is enough, we SHOULD encourage validators and full nodes to set snapshot enabled. 
+As the result, making it enabled by default can be more convenient to who wants to provide the snapshot to the network. 
```

Comment:
```suggestion
However, state syncing a node is almost impossible now since there are too few snapshot in the network. In order to ensure the number of snapshots is enough, we SHOULD encourage validators and full nodes to enable the snapshot creation process. 
As a result, enabling it by default can be more convenient to who wants to provide the snapshot to the network. 
```